tests/test_common.py

```diff
@@ -138,3 +138,11 @@ def test_lattice_shift_text():
 
     tables = camelot.read_pdf(filename, line_size_scaling=40, shift_text=['r', 'b'])
     assert df_rb.equals(tables[0].df)
+
+    
+def test_repr():
+    filename = os.path.join(testdir, "foo.pdf")
+    tables = camelot.read_pdf(filename)
+    assert repr(tables) == "<TableList n=1>"
+    assert repr(tables[0]) == "<Table shape=(7, 7)>"
+    assert repr(tables[0].cells[0][0]) == "<Cell x1=120.47812499999999 y1=218.41500000000002 x2=164.64 y2=233.89125>"
```

Comment:
This line is unpredictable, I guess.

Reply:
I'm removing this line.

Reply:
@vinayak-mehta Have you seen this?

Reply:
@vaibhavmule Why was it unpredictable? Because of the floating points? What if we round them to 2 points inside Cell's __repr__?

Reply:
Check: https://travis-ci.org/socialcopsdev/camelot/builds/437584814